backend/api/appsettings.Local.json

```diff
@@ -22,5 +22,11 @@
     "Notifications": {
       "ThrowExceptions": true
     }
+  },
+  "Keycloak": {
+    "Authority": "http://localhost:8080/auth/realms/pims",
```

Comment:
This makes it simpler to run the API locally without having to manually change this in your .env file.